src/Pages/Withdrawals.razor

```diff
@@ -165,23 +166,30 @@
                             }
                         </DisplayTemplate>
                         <EditTemplate>
-                            @if (!_isAmountDisabled)
-                            {
-                                <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args, _isAmountDisabled)">
-                                    <NumericPicker TValue="decimal" Disabled="_isAmountDisabled" Value="@((decimal)context.CellValue)" ValueChanged="(value) => { context.CellValue = value; }" CurrencySymbol="₿ " Max="@(_maxWithdrawal)" Min="_minWithdrawal" Decimals="8">
-                                        <Feedback>
-                                            <ValidationError />
-                                        </Feedback>
-                                    </NumericPicker>
-                                    <FieldHelp>
-                                        @($"Current amount: {Math.Round(PriceConversionService.BtcToUsdConversion((decimal)context.CellValue, _btcPrice), 2)} USD")
-                                    </FieldHelp>
+                                <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args)">
+                                    <NumericPicker TValue="decimal" @bind-Value="@_amount" Step="0.00001m" CurrencySymbol="₿ " Max="@(_maxWithdrawal)" Min="_minWithdrawal" Decimals="8" />
+                                     <FieldHelp>
```

Comment:
In the Withdrawals modal, when you have UTXOs selected manually, the numeric input should be disabled, and then enabled again when you click on "Default Coin Selection"

<img width="522" alt="Screenshot 2023-10-05 at 10 08 58" src="https://github.com/Elenpay/NodeGuard/assets/39995243/de50a077-ee04-40a9-9023-1361b422c9a0">

Reply:
Also if you click on "Withdraw all funds", the amount and the select coin buttons should be disabled, and if coins were selected previously, then it should remove the selection

<img width="506" alt="Screenshot 2023-10-05 at 10 15 20" src="https://github.com/Elenpay/NodeGuard/assets/39995243/545714bf-0820-4a97-aed7-716fd406554d">

Reply:
Good points, Rodrigo!